src/Types.hs

```diff
@@ -243,6 +243,10 @@ data IdeaJuryResultValue
 
 type instance Proto IdeaJuryResult = IdeaJuryResultValue
 
+ideaJuryResultValueToType :: IdeaJuryResultValue -> IdeaJuryResultType
+ideaJuryResultValueToType NotFeasible{} = IdeaNotFeasible
+ideaJuryResultValueToType Feasible{}    = IdeaFeasible
```

Comment:
if you only need this function in the story interpreter, would it make sense to define it there?